src/main/java/com/mojang/brigadier/builder/ArgumentBuilder.java

```diff
@@ -11,16 +11,25 @@
 
 import java.util.Collection;
 import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
 import java.util.function.Predicate;
 
 public abstract class ArgumentBuilder<S, T extends ArgumentBuilder<S, T>> {
+    private static final Predicate<?> TRUE = __ -> true;
```

Comment:
Can move this in a static method like 
```java
private static <T> Predicate<T> alwaysTrue() {
    return t -> true;
}
```
This prevents unchecked warnings.
Due to how java handles lambda like this, in fact, only one instance of the predicate will be ever created (instead of creating a new object every call)